docs/admin/researcher-setup/email-messaging.md

```diff
@@ -0,0 +1,139 @@
+|Failed scheduling pod|`Pod`|`FailedScheduling`|binding a pod to a node failed|2.9 and above|Pod, Job, Project, Namespace, User|
+You can configure the notifications service to listen to other Kubernetes events and send it via email by using the relevant `Kind` and event `Reason`.
+## Installation
+
+Install the notification service using the following commands:
+
+1. Set the helm repo to point to the notification service package using the following command:
+
+```
+helm repo add runai-notifications-service https://storage.googleapis.com/runai-notifications-service/
+
+helm repo update
+```
+
+2. Check for the latest version using the following command
+
+```
+helm repo search runai-notifications-service 
+
+```
+
+3. Install the latest version using the following command:
+
+```
+helm install runai-notifications-service/notifications-service --version 0.0.0
+```
+
+Note:
+
+You can change the service configuration values with the -f flag or edit them after installation.
+
+## Configuration
+
+The notification service is configured using a `configmap` file. The following is an example of a `configmap` file. Each of the tables below references a section in the `configmap` file.
+
+### `service` configuration
+
+This section defines the number of events that will be sent by the service. Use the following table to configure options in the `service` section of the `configmap` file.
+
+|Component|Field|description|default|
+|:----|:----|:----|:----|
+|`service`|`service.concurrent_limit`|maximum number of events the service will handle in parallel|50|
+|`service`|`service.cached_events`|queue size for events before blocking the listener|1000|
+
+### `listener` configuration
+
+This section defines the objects and events that the service will listen to and send as notifications. Use the following table to configure options in the `listener` section of the `configmap` file.
+
+| Component | Field | description | default |
+| --- |  --- |  --- |  --- |
+| `kubelistener` | `listener.relevant_objects` | white list of Kubernetes components for notifications | relevant_objects: <br> `kind:` <br>    `Podreasons:UnschedulableScheduled` <br><br> `kind:` <br>`PodGroupreasons: -Evict` |
+| `kubelistener` | `listener.relevant_namespaces` | white list of namespaces that the service should listen to for events (regex) | `runai-.*` |
+
+### `enrich` configuration
+
+Use the following table to configure options in the `enrich` section of the `configmap` file.
+
+| Component | Field | description | default |
+| --- |  --- |  --- |  --- |
+|`KubeMetadata`|`enricher.kubeMetadata.lables`|maps the event related object labels to specific properties in the output event | `release: workloadDisplayName` <br><br>`training.kubeflow.org/job-name: workloadDisplayName` <br><br>`serving.knative.dev/service: workloadDisplayName` <br><br>`project: project`|
```

Comment:
```suggestion
|`KubeMetadata`|`enricher.kubeMetadata.lables`| for internal use only - keep the default values | `release: workloadDisplayName` <br><br>`training.kubeflow.org/job-name: workloadDisplayName` <br><br>`serving.knative.dev/service: workloadDisplayName` <br><br>`project: project`|
```

Reply:
I will move this to a note at the top or bottom of the table.